src/tab.rs

```diff
@@ -1301,6 +1307,7 @@ pub struct Item {
     pub selected: bool,
     pub highlighted: bool,
     pub overlaps_drag_rect: bool,
+    pub size: u64,
```

Comment:
Consider using Option<u64> here so it can be determined if the size is still calculating or it is actually 0.